app/collections/manager.cjsx

```diff
@@ -17,7 +17,12 @@ module?.exports = React.createClass
     subject: React.PropTypes.object
     project: React.PropTypes.object
 
+  logModalChange: (clickedItem) ->
+    @props.contextRef?.geordi?.logEvent
```

Comment:
Why is Geordi passed in via props here when it's passed by context everywhere else?